Add assets and style html

simplegoose opened the pull request:
- Style the project's HTML skeleton.
- Add assets: fonts, images, and vectors to the project.
- Import the assets to HTML and CSS files.
- Update README.md

simplegoose committed:
On assets/fonts
- Add poppins-font into the project

simplegoose committed:
On css/style.css
- Add styles for each html skeleton for mobile view

simplegoose committed:
On index.html
- Add text to logo div in place of logo
- Background filter for nav and wrapper to wrap nav-items and close
- Add template text to html skeleton
- Import image assets into img tags
- Add button get my resume

simplegoose committed:
On css/style.css
- Add new color var for background filter
- Use new color var on background filter
- Change blur to 20px

simplegoose committed:
On assets/images/mobile
- New files to be used on mobile view, constisting of images

simplegoose committed:
On assets/images/vectors
- New vector images to be used on website

simplegoose committed:
On README.md
- Add readme file to project

simplegoose committed:
Merge branch 'add-html-file' into add-assets-and-style-html
Merge because previous branch didn't have readme file

simplegoose committed:
On index.html
- Comment out contact form
- Change link rel:css path to /css/style.css

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
```
Good job @simplegoose 👍, kindly link your CSS file properly. Right now your styling is not reflecting on your HTML template. ☺️
<img width="376" alt="image" src="https://user-images.githubusercontent.com/73854999/205343840-d83fcc19-336c-4c77-baaf-71b3508cfb46.png">

Alfredbis29 replied:
I can see that you made some changes but your style still not reflecting on your Html , kindly fix this for your project to look attractive :wink:  this is what we are seeing when run your project :point_down: 
![Screenshot from 2022-12-03 20-01-49](https://user-images.githubusercontent.com/88894525/205455856-cec6bc7d-e855-4c0c-9084-bd5f1cfba352.png)

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+            <button type="submit" class="buttons">Get In Touch</button>
+          </form>
+        </div>
+      </section> -->
```
Kindly remove unused code from your code in order to make your code base clean 🙏.

MaganaAsiati reviewed (approved):
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @simplegoose   , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


![clap](https://media0.giphy.com/media/j2GPljpvL8Lulm3d0X/giphy.gif)

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] [OPTIONAL] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <div class="logo">My Logo</div>
```
Great job so far 👏, but I think it will be better if your logo is wrapped in an `a` tag. This will allow your users to navigate back to the home page by clicking on the logo ☺️ .

simplegoose commented:
Hello @MaganaAsiati 
Thank you for your review, I will improve on how I write my commit messages.

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+  <div class="wrapper">
+    <header class="header">
+      <div class="logo">My Logo</div>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
```
Great job adding the hamburger👍. Kindly wrap your hamburger menu in a `nav` tag since the purpose of this section is to provide navigation links. It will be semantically correct if it is placed in a nav tag. 😎⚔️

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+    <header class="header">
+      <div class="logo">My Logo</div>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
```
Great job actualizing the hamburger with the `divs` but I think it will be better if you make use of an icon or image, this will make your code dry. 😊

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+              <img src="/assets/images/mobile/Snapshoot Portfolio-4.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Tonic</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
```
Instead of using span here, I think it will be better to make use of `li` wrapped in the `ul` tag since we are working with lists here. 😎

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+              <img src="/assets/images/mobile/Snapshoot Portfolio-1.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
```
Instead of using span here, I think it will be better to make use of `li` wrapped in the `ul` tag since we are working with lists here. 😎

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+              <img src="/assets//images//mobile//Snapshoot Portfolio-2.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Tonic</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
```
Instead of using span here, I think it will be better to make use of `li` wrapped in the `ul` tag since we are working with lists here. 😎

Lizdev-05 commented on index.html:
```diff
@@ -0,0 +1,217 @@
+              <img src="/assets/images/mobile/Snapshoot Portfolio-3.png" alt="" class="project-img">
+            </figure>
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <div class="explaining-spans">
+              <span>CANOPY</span>
+              <span></span>
+              <span>Back end dev</span>
+              <span></span>
+              <span>2015</span>
+            </div>
```
Instead of using span here, I think it will be better to make use of `li` wrapped in the `ul` tag since we are working with lists here. 😎

Alfredbis29 replied:
- [ OPTIONAL]AS required from the previous review you should use `li` wrapped in the `u;` tag as you are working with lists :wink:

Lizdev-05 reviewed (changes requested):
Hi @simplegoose, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
<img src="https://media2.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>
## Highlights
- Good commit history 💰
- Good PR title/description 💯
- All linters passed 🏅
- Personalised README file 👍

## Required Changes ♻️

- [x] Kindly remove/close unused pull requests so as to avoid confusion
- [x] Kindly check your style sheet, it is not linked to your HTML file correctly and the stylings are not taking effect now.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Alfredbis29 reviewed (changes requested):
Hi @simplegoose 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
![](https://camo.githubusercontent.com/ea5a41c13a43074321873dea3237d149557ac13d29ccf550ac00b20bdfc39068/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

## Suggested changes

_Check the comments under the review._

### OPTIONAL CHANGES :brown_circle: 

- [x]  check under the review

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

simplegoose commented:
Hello @Alfredbis29 
I think you reviewed the wrong pull request.

simplegoose committed:
Add icon-menu asset

simplegoose committed:
Style anchor tag
Style hamburger menu
Style newly added list for project list

simplegoose committed:
Change file source path
Wrap logo inside anchor tag
Change card explaining span to list
Uncomment code
Use icon-menu img as hamburger-menu

simplegoose committed:
Lint index.html and style.css

simplegoose committed:
Merge branch 'main' into add-assets-and-style-html
